src/client/Client.cc

```diff
@@ -526,7 +532,8 @@ void Client::_finish_init()
     lderr(cct) << "error registering admin socket command: "
 	       << cpp_strerror(-ret) << dendl;
   }
-  ret = admin_socket->register_command("mds_sessions",
+  ret = admin_socket->register_command("mds_sessions "
+		                       "name=cap-dump,type=CephChoices,strings=--cap-dump,req=false",
```

Comment:
```suggestion
		                       "name=cap-dump,type=CephBool,req=false",
```

Reply:
@batrick What did I miss? `Invalid command JSON` is returned when the type is changed from `CephChoices` to `CephBool`.

Reply:
@batrick fix validate type CephBool cause 'invalid command json'

https://tracker.ceph.com/issues/46985

Reply:
great work!